### DISCUSSION TITLE
Using site: parameter doesn't return correct number of indexed pages

### DISCUSSION THREAD

**Discussion by VictorAus** [eliasdabbas/advertools#168] [Q&A]
Hello,

I am trying to get the number of indexed pages from a bunch of websites that show up as linking back to our website in Google Search Console. 

We want to identify low-quality websites, one of the criteria is by looking at how many pages are indexed in Google. A low number of pages or a sudden drop in the number of indexed pages could be a red flag and we would want to Disavow those domains.

When I perform the "site:domain" search the number of indexed pages does not match with a normal Google search. I checked location and language settings and played around as much as I could with the settings but nothing helped.

Here are screenshots:
https://prnt.sc/1uvt9t1
https://prnt.sc/1uvtwil
https://prnt.sc/1uvu27e

Please can someone point me in the right direction?

Thanks

**Comment by eliasdabbas** [comment]
Good catch!

I'm not sure why Google returns a different number in the API than the one on the regular search.

I'll see if I can figure it out, and they also have a community forum that might help: 

https://support.google.com/websearch/community?hl=en

**Comment by VictorAus** [comment]
I posted there as well and they referred to me this link https://support.google.com/programmable-search/answer/70392?hl=en
Looks like it is not designed to be used with Site: parameter

**Reply by eliasdabbas** [reply]
Thanks a lot for checking.
Although it's not what I hoped, this is still a good reference link for anyone else interested in knowing more about the functionality of the CSE, and some caveats/limits.
